src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Sami32 If you have some samples and some ffmpeg command line options for me, I could test transcode some and see how fast/slow it is. If most CPU's can't do it, it isn't much point in implementing it, is it?